.github/workflows/lint.yaml

```diff
@@ -67,6 +67,11 @@ jobs:
       - name: Run Helm linter
         run: |
           helm lint ${{ env.HELM_CHARTS_DIR }}
+      - name: Run TypeSpec format check
+        run: |
+          pushd typespec
+          npx tsp format --check "**/*.tsp"
```

Comment:
Will this actually fail the step if it encounters an error? You might need `set -e`

Reply:
![image](https://github.com/radius-project/radius/assets/5220939/651b4bef-6b54-4a50-80c7-bc0607c99d24)

Reply:
Great